python/hsfs/feature_view.py

```diff
@@ -3640,7 +3643,11 @@ def delete_log(self, transformed: Optional[bool]=None):
          # Raises
              `hsfs.client.exceptions.RestAPIError` in case the backend fails to delete the log.
          """
-        return self._feature_view_engine.delete_feature_logs(self, transformed)
+        if self._feature_logging is None:
+            self._feature_logging = self._feature_view_engine.get_feature_logging(self)
```

Comment:
What if `delete_log` is called in a new FV without feature_logging enabled? (no call to `enable_feature_logging()` or `log()`) ?
I guess this line will return None, and then fail in the `delete_feature_logs()` ?